.gitignore

```diff
@@ -0,0 +1,3 @@
+/.idea/
+/spring-graphql-api/src/main/docker/.env
+/spring-graphql-api/.env
```

Comment:
We should probably mention ignoring the .env files. It's automatically happens for the Next app.